tests/fixtures/windows-10/dir-Q.out

```diff
@@ -0,0 +1,21 @@
+ Volume in drive C has no label.
+ Volume Serial Number is 246B-3B5A
+
+ Directory of C:\Program Files\Internet Explorer
+
+03/24/2021  03:15 PM    <DIR>          NT SERVICE\TrustedInsta.
+03/24/2021  03:15 PM    <DIR>          NT SERVICE\TrustedInsta..
+12/07/2019  02:49 AM    <DIR>          NT SERVICE\TrustedInstaen-US
+12/07/2019  02:09 AM            54,784 NT SERVICE\TrustedInstaExtExport.exe
+03/24/2021  03:15 PM                 0 BUILTIN\Administrators file name.txt
+12/07/2019  02:09 AM            54,784 NT SERVICE\TrustedInstahmmapi.dll
+12/07/2019  02:09 AM           515,072 NT SERVICE\TrustedInstaiediagcmd.exe
+12/07/2019  02:09 AM           504,832 NT SERVICE\TrustedInstaieinstal.exe
+12/07/2019  02:09 AM           224,768 NT SERVICE\TrustedInstaielowutil.exe
+12/07/2019  02:09 AM           421,888 NT SERVICE\TrustedInstaIEShims.dll
+12/06/2019  02:47 PM           819,136 NT SERVICE\TrustedInstaiexplore.exe
+12/07/2019  02:14 AM    <DIR>          NT SERVICE\TrustedInstaimages
+12/07/2019  02:14 AM    <DIR>          NT AUTHORITY\SYSTEM    SIGNUP
+12/07/2019  02:09 AM            48,536 NT SERVICE\TrustedInstasqmapi.dll
```

Comment:
The `dir /Q` output looks quite hard, if not impossible, to parse correctly. Maybe we should just specify that `/Q` is not supported or just don't include it as one of the supported options. People can still try to use it at their own risk, but it won't be 'supported'